sagemaker-training-compiler/huggingface/pytorch_single_gpu_single_node/bert-base-cased/bert-base-cased-single-node-single-gpu.ipynb

```diff
@@ -81,7 +81,7 @@
    "metadata": {},
```

Comment:
<p>Do we need to install torch 1.12 instead of torch 1.13.1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3812/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Yes. You are right, we don't need to install torch here. I will remove this for all of three notebooks.